examples/galton.jl

```diff
@@ -106,6 +107,7 @@ function galton(seed)
     height = 350
     offset = width / (n - 1)
     radius = offset / 4
+    p = 0.5
```

Comment:
Do you mind having this as a parameter like `seed`?

Reply:
I forgot about this previously, updated it now